src/utils/objectHas.js

```diff
@@ -1,5 +1,5 @@
 export default (obj, keys) => {
-  if (typeof obj !== 'object') {
+  if (typeof obj !== 'object' || typeof obj === 'null') {
```

Comment:
It seems that `typeof` (according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof) won't ever 'null'. Weirdly enough, since `typeof null === 'object'`, I think you meant `obj === null`? It is indeed a good check :-) Thanks

Reply:
Sorry, I didn't check carefully when I submitted it, and I didn't seriously think about it when I wrote it. Such an implementation is indeed problematic, because `typeof null === 'object'`

Reply:
I will reintroduce a commit to fix this problem.

Reply:
👍